package.json

```diff
@@ -9,16 +9,17 @@
 		"lint": "next lint"
 	},
 	"dependencies": {
-		"@next/font": "13.0.7",
+		"@cloudscape-design/components": "^3.0.171",
```

Comment:
Ahh, yeah, noticed this before but disregarded. Looks like there's no official support for React 18 yet, but they are working on it: cloudscape-design/components#271. Don't think I've noticed any major issues specifically because of this.